src/main/java/org/team1540/robot2025/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,91 @@
+package org.team1540.robot2025.subsystems.intake;
+
+import com.ctre.phoenix6.configs.MotionMagicConfigs;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.Slot1Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class IntakeIOReal implements IntakeIO {
+
+    // rotation of horizontal beams for the intake, clockwise to intake, counter-clockwise to spit out
+    private final TalonFX spinFalcon = new TalonFX(IntakeConstants.SPIN_FALCON_ID);
+
+    // controls position of intake
+    private final TalonFX pivotFalcon = new TalonFX(IntakeConstants.PIVOT_FALCON_ID);
+
+    private final MotionMagicVoltage motorRequest = new MotionMagicVoltage(0).withSlot(0);
+
+    // clockwise to intake, counter-clockwise to spit out
+    private final SparkMax neo = new SparkMax(IntakeConstants.NEO_ID, SparkLowLevel.MotorType.kBrushless);
+
+    public IntakeIOReal() {
+
+        TalonFXConfiguration spinTalonFXConfigs = new TalonFXConfiguration();
+        TalonFXConfiguration pivotTalonFXConfigs = new TalonFXConfiguration();
+
+        spinTalonFXConfigs.CurrentLimits.withStatorCurrentLimitEnable(true);
+        spinTalonFXConfigs.CurrentLimits.withStatorCurrentLimit(IntakeConstants.SPIN_FALCON_CURRENT_LIMIT);
+        spinTalonFXConfigs.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+        spinTalonFXConfigs.MotorOutput.NeutralMode = NeutralModeValue.Brake;
```

Comment:
You want to be able to spin the rollers of the intake by hand, so it should be set to coast.